excalidraw-app/vite-docker.config.mts

```diff
@@ -0,0 +1,17 @@
+import { defineConfig, mergeConfig, loadEnv } from "vite";
+import viteConfig from "./vite.config.mts";
```

Comment:
@ad1992 do we need to write vite configs in typescript (we can style typecheck them in vscode)? Or what's the workaround for this other than using `mjs` files.